rig.js

```diff
@@ -59,6 +69,23 @@ const localRaycaster = new THREE.Raycaster(); */
   return geometry;
 })(); */
 const _makeRig = app => {
+
+
+    function initWindParameters(rig) {
+        var hairPhysics = rig.app.getComponent('hairPhysics');
```

Comment:
Hair physics is not the best name, since all avatars should have hair physics. This is about wind.